Makefile

```diff
@@ -59,9 +71,8 @@ endef
 define run_devnet
     docker run $(TTY_SETTING) $(INTERACTIVE_SETTING) --rm \
     --dns 8.8.8.8 \
-    -m $(MAX_RAM) \
-    --cpus="16" \
-    --oom-kill-disable \
+    $(RAM_OPT) \
```

Comment:
In the case that we could find `MAX_RAM` the option is completely removed now.
We should be setting `RAM_OPT` in that case too.

Reply:
I tested it locally on linux and macOS, and it seems to work well.
Linux:
<img width="1679" alt="image" src="https://user-images.githubusercontent.com/10941036/122769698-2b0cc680-d27b-11eb-909a-055950c03288.png">